src/status_im/contexts/onboarding/events.cljs

```diff
@@ -56,7 +56,9 @@
               {:dispatch       [:navigate-to-within-stack
                                 [:screen/onboarding.generating-keys :screen/onboarding.new-to-status]]
                :dispatch-later [{:ms       constants/onboarding-generating-keys-animation-duration-ms
-                                 :dispatch [:onboarding/navigate-to-identifiers]}]
+                                 :dispatch [:navigate-to-within-stack
+                                            [:screen/onboarding.enable-notifications
+                                             :screen/onboarding.new-to-status]]}]
```

Comment:
There's a `:onboarding/navigate-to-enable-notifications` event for this after the onboarding "simplification" PR https://github.com/status-im/status-mobile/pull/20386, and you need to pass the from screen for the navigation stack to this event

Because after that PR, user can now navigate to create profile screen (and all the following screen) from either `:screen/onboarding.sync-or-recover-profile` or `:screen/onboarding.new-to-status`

Reply:
thanks @yqrashawn let me rebase my code and adjust